frontend/app/templates/components/public-schools/summary.hbs

```diff
@@ -126,22 +126,18 @@
         With Action
       </td>
       <td>
-        {{futureConditions.enrollWithAction}}
```

Comment:
do these need to be replaced with something else?

Reply:
Yep! Working on another commit that will address this and a few other issues that should come along with this PR.